doc/DeepSpeech.md

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
@reuben Maybe that's why @tilmankamp took that approach. I don't know. Also, he's on holiday this week. So we can't ask him.